osu.Game/Overlays/Chat/ChatLine.cs

```diff
@@ -62,6 +62,42 @@ private Color4 getUsernameColour(Message message)
             return username_colours[message.UserId % username_colours.Length];
         }
 
+        private string getUsernameDisplayString(Message message)

```

Comment:
all these methods are overkill and shouldn't be required. just inline the changes